app/src/ui/app-menu/menu-pane.tsx

```diff
@@ -91,7 +91,13 @@ interface IMenuPaneProps {
   readonly onClearSelection: (depth: number) => void
 
   /** The id of the element that serves as the menu's accessibility label */
-  readonly ariaLabelledby: string
+  readonly ariaLabelledby?: string
+
+  /** Whether we move focus to the next menu item with a label that starts with
+   * the typed character if such an menu item exists. */
+  readonly allowFirstCharacterNavigation?: boolean
```

Comment:
I flagged this as to not introduce any new unexpected behavior to the windows app menu.. tho I am not sure it is a negative thing.. but they already have accessor keys defines so probably would be conflicting.